src/dials/command_line/import_xds.py

```diff
@@ -125,14 +154,43 @@ def __call__(self, params, options):
         table["id"] = flex.int(len(hkl), 0)
         table["panel"] = panel
         table["miller_index"] = hkl
+
         table["xyzcal.px"] = xyzcal
         table["xyzobs.px.value"] = xyzobs
-        table["intensity.cor.value"] = iobs
-        table["intensity.cor.variance"] = flex.pow2(sigma)
+        # uncorrected "partials":
         table["intensity.prf.value"] = iobs * peak / rlp
         table["intensity.prf.variance"] = flex.pow2(sigma * peak / rlp)
-        table["lp"] = 1.0 / rlp
+        table.set_flags(flex.bool(table.size(), True), table.flags.integrated_prf)
+
         table["d"] = flex.double(uc.d(h) for h in hkl)
+
+        table["partiality"] = peak
+        table["profile.correlation"] = corr
+
+        table.centroid_px_to_mm(self._experiments)
+
+        # same as prf?
+        table["intensity.sum.value"] = iobs * peak / rlp
+        table["intensity.sum.variance"] = flex.pow2(sigma * peak / rlp)
+        table.map_centroids_to_reciprocal_space(self._experiments)
+        table["intensity.scale.value"] = flex.double(table.size(), 1.0)
```

Comment:
In testing this PR, I found that having the `"intensity.scale.value"` column present caused some issues, as some programs (e.g. `dials.symmetry`) use this to assume that the data was scaled, which then caused issues due to the zero "intensity.scale.variance". It would be safer to remove the three columns `"intensity.scale.value"`, `"intensity.scale.variance"` and `"inverse_scale_factor"`, to make the output conform to the expected form of an `integrated.refl` file.

Reply:
I'm not 100% sure, but I might have added those because they are needed further down in the computation of ZETA and LP?

Should we zap them afterwards once everything is done? Or not create them at all in the first place?

Reply:
I have just tested your branch without those three columns being created and it seems to work fine, I wouldn't expect any further functions to need them in this script, so I think they can just be left out, unless you can reproduce a failure due to their absence?